### PR TITLE
[Merged by Bors] - Book spelling and grammar corrections

### PR DESCRIPTION
**Pull request by kausikds**
## Issue Addressed

There are few spelling and grammar errors in the book.

## Proposed Changes

Corrected those spelling and grammar errors in the below files
- book/src/advanced-release-candidates.md
- book/src/advanced_networking.md
- book/src/builders.md
- book/src/key-management.md
- book/src/merge-migration.md
- book/src/wallet-create.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Lighthouse is:
 - Security-focused. Fuzzing techniques have been continuously applied and several external security reviews have been performed.
 - Built in [Rust](https://www.rust-lang.org), a modern language providing unique safety guarantees and
 	excellent performance (comparable to C++).
-- Funded by various organisations, including Sigma Prime, the
+- Funded by various organizations, including Sigma Prime, the
```
Comment (michaelsproul) [inline README.md]:
```suggestion
- Funded by various organisations, including Sigma Prime, the
```

Most of us are based in Australia, so we're committed to a mix of UK and US English. We hope our US audience understand anyway ;)

File: book/src/advanced-release-candidates.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ ergonomics of new features or changes.
 Github will clearly show such releases as a "Pre-release" and they *will not* show up on
 [sigp/lighthouse/releases/latest]. However, release candidates *will* show up on the
 [sigp/lighthouse/releases] page, so **please pay attention to avoid the release candidates when
-you're looking for stable Lighthouse**.
+you're looking for a stable Lighthouse**.
```
Comment (michaelsproul) [inline book/src/advanced-release-candidates.md]:
```suggestion
you're looking for stable Lighthouse**.
```

Stable here refers to the `stable` branch, perhaps we could say "the stable version of Lighthouse".

**Review by michaelsproul — CHANGES_REQUESTED** [review]
Thanks!

**Review by kausikds — COMMENTED** [review]
I reviewed the rest of the changes and found those are the correct changes to make.

**Comment by michaelsproul** [comment]
@iKausik Did you intend to push another commit? It looks like you just marked the feedback as resolved without changing anything

**Review by michaelsproul — APPROVED** [review]

**Comment by michaelsproul** [comment]
Thanks! Will merge this after v3.2.1 has shipped

**Pull request closed without merging**